backend/pkg/report/report.go

```diff
@@ -30,10 +30,15 @@ func NewReport(logger *slog.Logger, cfg *config.Config, version *version.Version
 	u, _ := url.Parse(raw)
 	client := request.NewClient(u.Scheme, u.Host, 30*time.Second)
 
+	// 从配置文件读取机器ID文件路径
+	idFilePath := cfg.DataReport.MachineIDFile
+	if idFilePath == "" {
```

Comment:
建议将默认路径定义为常量，以提高代码的可维护性。

```suggestion
const defaultMachineIDFile = "/app/static/.machine_id"

// ...

idFilePath := cfg.DataReport.MachineIDFile
if idFilePath == "" {
    idFilePath = defaultMachineIDFile
}